animations/hero-animations/index.md

```diff
@@ -0,0 +1,670 @@
+---
+
+<div class="whats-the-point" markdown="1">
+
+<b> <a id="whats-the-point" class="anchor" href="#whats-the-point" aria-hidden="true"><span class="octicon octicon-link"></span></a>What you'll learn:</b>
+
+* Create a Hero animation using Flutter's Hero widget.
+* Fly the hero from one screen to another.
+* Animate the transformation of a hero's shape from circular to
+  rectangular while flying it from one screen to another.
+* Hero animations are sometimes called _shared element transitions_
+  or _shared element animations_.
+</div>
+
+You've probably seen hero animations many times. For example,
+a screen displays a list of thumbnails representing items
+for sale.  Selecting an item flies it to a new screen,
+containing more details and a "Buy" button. Flying an image
+from one screen to another is called a _hero animation_
+in Flutter, though the same motion is sometimes referred to as
+a _shared element transition_.
+This guide demonstrates how to build standard
+hero animations, and hero animations that transform the
+image from a circular shape to a square shape during flight.
+
+* TOC Placeholder
+{:toc}
+
+<aside class="alert alert-info" markdown="1">
+**New to Flutter?**
+You should first understand how to create a layout using Flutter’s
+widgets.  For more information, see [Building Layouts in
+Flutter](/tutorials/layout/).
+</aside>
+
+<aside class="alert alert-info" markdown="1">
+**Terminology:**
+A [_Route_](/routing-and-navigation/) describes a page or screen
+in a Flutter app.
+</aside>
+
+You can create this animation in Flutter with Hero objects.
+When a new route appears, a hero animates from the current route to
+the new one. Typically, heroes are small parts of the UI, like
+images, that both routes have in common. From the user's
+perspective the hero "flies" between the routes.
+This guide shows how to create the following hero animations:
+
+<ul>
+<li markdown="1">Standard hero animations<br>
+A _standard hero animation_ flies the hero from one route to a new
+route, usually landing at a different location and with a
+different size.
+
+The following video (recorded at slow speed) shows a typical example.
+Tapping the flippers in the center of the route flies them to the
+upper left corner of a new, blue route, at a smaller size.
+Tapping the flippers in the blue route
+(or using the device's back-to-previous-route gesture)
+flies the flippers back to the original route.
+
+<center><iframe width="560" height="315" src="https://www.youtube.com/embed/uF7_JfhBsww" frameborder="0" allowfullscreen></iframe></center>
+</li>&nbsp;
+
+<li markdown="1">Radial hero animations<br>
+In _radial hero animation_, as the hero flies between routes
+its shape appears to change from circular to rectangular.
+
+The following video (recorded at slow speed),
+shows an example of a radial hero animation. At the start,
+a row of three images appears at the bottom of the route.
+Tapping any of the circles flies that image to a new route
```

Comment:
circles => circular images I guess. It seems important to use the same term to refer to the demo's heroes.